### PR TITLE
Compute town with districts population

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: france.py [hunk 1]
```diff
@@ -423,6 +423,33 @@ def town_with_districts(db, filename):
     success('Attached {0} districts to Lyon'.format(result.modified_count))
 
 
+@town.postprocessor()
+def compute_town_with_districts_population(db, filename):
+    info('Computing Paris town districts population')
+    districts = db.find({'_id': {'$in': PARIS_DISTRICTS}})
```
Comment (davidbgk) [inline france.py]: I'm pretty sure there is a one-magic-mongo-command to do that but given that it takes less than 1 sec per town and that I spent more that 30 minutes on it, that doesn't seem worth it :smiling_imp:

Comment (noirbizarre) [inline france.py]: Yes, there is [aggregation](https://docs.mongodb.org/manual/reference/operator/aggregation/) and [`$sum` operator](https://docs.mongodb.org/manual/reference/operator/aggregation/sum/) but as you said, it's not worth id given the amount of data to handle here.

File: geo.py [hunk 2]
```diff
@@ -173,9 +173,11 @@ def build_aggregate(self, code, name, zones, properties, db):
         data.update(properties)
         return data
 
-    def postprocess(self, workdir, db):
+    def postprocess(self, workdir, db, only=None):
         '''Perform postprocessing'''
         for url, processor in self.postprocessors:
+            if only is not None and processor.__name__ != only:
```
Comment (noirbizarre) [inline geo.py]: nice :)

**Comment by noirbizarre** [comment]
r+

**Pull request merged**